Aspire.Hosting.* READMEs (#3828)
microsoft/aspire: src/Aspire.Hosting.Azure.AppConfiguration/README.md

dotnet add package Aspire.Hosting.Azure.AppConfiguration
```

## Configure Azure Provisioning for local development

> Adding Azure resources to the .NET Aspire application model will automatically enable automated provisioning
> for Azure resources so that you don't need to configure them manually. Provisioning requires a number of settings

Reviewer: ```suggestion
Adding Azure resources to the .NET Aspire application model will automatically enable development-time provisioning
for Azure resources so that you don't need to configure them manually. Provisioning requires a number of settings
```

1. not sure we need to say "automatically enable automated"
2. We should be clear this is development-time only

to be available via .NET configuration. Set these values in user secrets in order to allow resources to be configured
automatically.

```json
{
    "Azure": {
      "SubscriptionId": "<your subscription id>",
      "ResourceGroup": "<your resource group name>",
      "AllowResourceGroupCreation": true,
      "Location": "<azure location>"
    }
}
```

> > NOTE: Note that developers must have Owner access to the target subscription so that role assignments

Reviewer: ```suggestion
> NOTE: Developers must have Owner access to the target subscription so that role assignments
```

                       .WithReference(appConfig);
```

> > NOTE: Consider setting the name of your resource to something other than "config" or "appconfig". Even though during deployment random suffix will be added it is still possible to get a name collision.

Reviewer: ```suggestion
> NOTE: Consider setting the name of your resource to something other than "config" or "appconfig". Even though during deployment a random suffix will be added it is still possible to get a name collision.
```
